spec/models/exercise_spec.rb

```diff
@@ -58,4 +57,40 @@
       end
     end
   end
+
+  describe '.create_zip' do
```

Comment:
The convention is to prepend instance method names with a hash e.g. `#create_zip`